lib/canvas/live_events.rb

```diff
@@ -101,6 +101,23 @@ def self.get_submission_data(submission)
     }
   end
 
+  def self.get_attachment_data(attachment)
+    {
+      attachment_id: attachment.global_id,
+      user_id: attachment.global_user_id,
+      display_name: LiveEvents.truncate(attachment.display_name),
+      filename: LiveEvents.truncate(attachment.filename),
+      context_type: attachment.context_type,
+      context_id: attachment.context_id,
+      content_type: attachment.content_type,
+      folder_id: attachment.folder_id,
+      unlock_at: attachment.unlock_at,
+      lock_at: attachment.lock_at,
+      updated_at: attachment.updated_at,
+      hidden: attachment.hidden
```

Comment:
For now, we should probably leave `hidden` off. The problem is that accessing `hidden` will load the entire folder parentage if it's not already loaded. That is likely not a problem (I suspect it's usually loaded), but we're trying to be super careful about only sending data in live events that we're sure is already loaded. Plus, this is breaking a spec. If you really need this attribute, we can dig in and make sure we're comfortable adding it, but the easiest way forward would be to just remove it.